### ISSUE TITLE
Comments in Nim are in the wrong format

### ISSUE THREAD

**Issue by lf-araujo** [chubin/cheat.sh#134]
Unless I misunderstood your API, results should have comments in the syntax  of the desired language. 

In the case of Nim, comments are Python-like (#), but results come in C format (/**/).

**Comment by chubin** [comment]
Yes, that is because Nim is not (yet) supported. Here is the list of the supported languages:

https://github.com/chubin/cheat.sh#programming-languages-cheat-sheets
